devtools/common/src/main/resources/templates/basic-rest/kotlin/build.gradle-template.ftl

```diff
@@ -1,20 +1,8 @@
-// this block is necessary to make enforcedPlatform work for Quarkus plugin available
-// only locally (snapshot) that is also importing the Quarkus BOM
-buildscript {
-    repositories {
-        mavenLocal()
-    }
-    dependencies {
-        classpath "io.quarkus:quarkus-gradle-plugin:${quarkusVersion}"
-    }
-}
-
 plugins {
     id 'kotlin'
+    id 'io.quarkus'
```

Comment:
cool! Does it mean the build plugin was finally deployed on the plugins portal?

Reply:
No it's not. I need to include that in the release process. I'll try to work on it early next week for the next release.

Reply:
Ok. So I guess the template is crafted like this so that it _will_ work after that?

Reply:
It should work. I tested it. One of the reasons we had the legacy way of applying the plugin was due to a bug in Gradle which was fixed in 5.5.1.